index.html

```diff
@@ -3874,7 +3874,58 @@ <h3>Syntactic Sugar</h3>
             </li>
           </ul>
         </section>
-
+        <section>
+          <h3>Lists and Arrays</h3>
+          <p>
+Lists and arrays, even lists of lists, are possible when using [[JSON-LD]] 1.1
+as this specification does. We encourage those who want RDF semantics with use
```

Comment:
```suggestion
Lists and arrays, even lists of lists, are possible when using [[JSON-LD]] 1.1. 
We encourage those who want RDF semantics with use
```

The "as this specification does" could be misinterpreted as "this specification uses lists, and arrays, and lists of lists". Attempting to clarify.

Reply:
One word changed beyond @msporny's suggestion
```suggestion
Lists and arrays, even lists of lists, are possible when using [[JSON-LD]] 1.1.
We encourage those who want RDF semantics in use
```